wdio.conf.ios.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv').config();
+
+const path = require('path');
+const { UtamWdioService } = require('wdio-utam-service');
+// use prefix 'DEBUG=true' to run test in debug mode
+const { DEBUG } = process.env;
+const EXPLICIT_TIMEOUT = 60 * 1000;
+const DEBUG_TIMEOUT = EXPLICIT_TIMEOUT * 30;
+
+exports.config = {
```

Comment:
I would suggest to declare a base config that export an object with all shared properties with IOS and Android and then merge this config with the specific options for each to avoid having duplicating config settings. I don't know if WDIO has some kind of `extends` or `overrides` built-in but a plain JS object merge would just work:

```js
{
   ...baseMobileConfig,
   iosMobileConfig
}
```

Reply:
updated